tests/Microsoft.AspNet.SignalR.Tests/TaskAsyncHelperFacts.cs

```diff
@@ -113,7 +113,7 @@ public void ContinuePreservedCulturePreservesCulture()
                 (task, continuation) => task.ContinueWithPreservedCulture(_ => continuation()));
         }
 
-        [Fact]
+        [Fact(Skip = "Started failing after upgrading xunit from 1.9.1 to 2.2.0")]
```

Comment:
OK. We actually saw exactly the same in SignalR-Server https://github.com/aspnet/SignalR-Server/blob/9680e9c2c185a19604390003a6f10489d0538a56/test/Microsoft.AspNetCore.SignalR.Server.Tests/TaskAsyncHelperFacts.cs#L127-L128. I once briefly looked at it but couldn't find anything obvious. Maybe I should ask xUnit folks if they have any ideas.